src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -224,6 +224,17 @@ trait WorkflowComponent {
       findWorkflowsBySubmissionId(submissionId).filter(_.status inSet(statuses.map(_.toString))).result
     }
 
+    def countWorkflowsByQueueStatus: ReadAction[Map[String, Int]] = {
+      findQueuedAndRunningWorkflows.result.map { recs =>
+        val grouped = recs.groupBy {
+          // lump all of the workflows currently being handled by Cromwell into "Active"
+          case runningRecord if WorkflowStatuses.runningStatuses.map(_.toString).toSet contains runningRecord.status => "Active"
```

Comment:
It's convenient to do so in Rawls because we have statuses partitioned by Queued / Running / Terminal in `WorkflowStatuses`.

I can see how this would be more extensible if we simply included every status and let callers/UI handle partitioning.  In that case I'd modify this further to include the Terminal statuses as well.